server/.env.docker

```diff
@@ -2,4 +2,5 @@ PORT=1000
 POSTGRES_URL=postgresql://prisma:prisma@postgres:5432/prisma?schema=pyrog
 APP_SECRET=deadbeefdeadbeefdeadbeefdeadbeef
 FHIR_API_URL=http://reverse-proxy/api
-REDIS_URL=redis://redis:6379
\ No newline at end of file
+REDIS_URL=redis://redis:6379
+ENV=test
```

Comment:
Do we want to keep that?

Reply:
yes, I don't think we want to setup the whole hydra mess when running locally in docker-compose (see #429)